docs/integrations/sources/postgres.md

```diff
@@ -74,10 +74,27 @@ GRANT SELECT ON TABLE <view_name> IN SCHEMA <schema_name> to <user_name>;
 ```
 
 This issue is tracked in [\#9771](https://github.com/airbytehq/airbyte/issues/9771).
+### <a name="jdbc-url-params"></a>3. (Advanced) configure additional JDBC URL parameters
+
+This is an advanced configuration option. Users are advised to use it with caution. If you need to customize the JDBC connection beyond common options you can specify additional JDBC URL parameters in `JDBC URL Params` field. The value of `JDBC URL Params` field
+should have the form of key-value pairs separated by the symbol `&`. 
+
+E.g. `key1=value1&key2=value2&key3=value3`
+
+These parameters will be added at the end of the JDBC URL that the AirByte will use to connect to your Postgres database.
 
-#### 3. Optionally, set up CDC. Follow the guide [below](postgres.md#setting-up-cdc-for-postgres) to do so.
+Do not use any of the following keys in `JDBC URL Params` field, because they will be overwritten by Airbyte:
```

Comment:
could we validate this in the check operation? (or maybe even as a regex in the connector spec?) Burying it in the docs seems like an easy way for users to not notice it.

Reply:
Good idea! I filed an issue for that: https://github.com/airbytehq/airbyte/issues/13851